### PR TITLE
File manager extension documentsDirectoryPath and cachesDirectoryPath (by @vilapuigvila)

### PR DESCRIPTION
**Pull request by Khalian**
## Checklist

- [x] New Extension
- [x] New Test
- [x] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by Khalian** [comment]
https://github.com/goktugyil/EZSwiftExtensions/pull/316

File: Sources/FileManagerExtension.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//
+
+import Foundation
+
+extension FileManager {
+    
+    /// EZSE: Returns path of documents directory
+    public var documentsDirectoryPath: String {
+        let paths = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true) as [String]
+        return paths[0]
```
Comment (piv199) [inline Sources/FileManagerExtension.swift]: Potential crash...

Comment (Khalian) [inline Sources/FileManagerExtension.swift]: @vilapuigvila You might want to take a look at the review comment.

Comment (vilapuigvila) [inline Sources/FileManagerExtension.swift]: @piv199 you're right ... @Khalian ok thank you

Comment (Khalian) [inline Sources/FileManagerExtension.swift]: I took care of it with an optional.

**Pull request merged**